### PR TITLE
General code improvements

### PR DESCRIPTION
**Pull request by Atmos4**
First round of changes to prepare for a massive club management PR

### CODE REVIEW

File: app/template/layout.php
Comment (Mathieu414) [inline app/template/layout.php:1]: keep start_intro

File: engine/core/Router.php [hunk 1]
```diff
@@ -10,7 +9,9 @@ private function render($path_to_include = "app/pages/404.php")
         try {
             $this->level = ob_get_level();
             ob_start();
-            include_once $path_to_include;
+            if (!@include $path_to_include) {
+                throw new Error("Router: $path_to_include not found");
+            }
```
Comment (Mathieu414) [inline engine/core/Router.php:14]: change to require

**Review by Mathieu414 — APPROVED** [review]
GG

File: app/template/layout.php
Comment (Atmos4) [reply app/template/layout.php:1]: Good catch, done

File: engine/core/Router.php (hunk 1 above)
Comment (Atmos4) [reply engine/core/Router.php]: Yes way simpler, thanks

**Pull request merged**